pkg/hintrunner/zero/zerohint_usort_test.go

```diff
@@ -13,15 +15,14 @@ func TestZeroHintUsort(t *testing.T) {
 		"UsortEnterScope": {
 			{
 				ctxInit: func(ctx *hinter.HintRunnerContext) {
-					err := ctx.ScopeManager.AssignVariable("__usort_max_size", feltUint64(1))
-					if err != nil {
-						t.Fatal(err)
-					}
+					ctx.ScopeManager.EnterScope(map[string]any{
+						"__usort_max_size": uint8(1),
```

Comment:
Why did you use uint8 instead of feltUint64 and created a case for uint8 in varValueInScope? More efficient ?

Reply:
It was unneccessary complication, went back to uint64. Performance in case of tests is not as important as readability.